Tests.Unit.Core.ApplicationServices/ApplicationServices/ItInterfaceServiceTest.cs

```diff
@@ -529,10 +525,410 @@ public void GetAvailableInterfaces_Returns_Public_Interfaces_If_User_Has_Access_
             Assert.Equal(expectedNumberOfInterfaces, result.Count());
         }
 
+        [Fact]
+        public void CreateNewItInterface_Returns_Created_Interface_With_User_Specified_Uuid()
```

Comment:
De to første tests kan laves med en parametriseret theory (parametrisering af optional params)